extensions/amp-autocomplete/0.1/test/test-amp-autocomplete.js

```diff
@@ -876,25 +922,21 @@ describes.realWin(
       expect(impl.selectItem_(disabledItem)).to.be.undefined;
     });
 
-    it('should not return disabled items from getEnabledItems_()', () => {
-      impl.templateElement_ = doc.createElement('template');
+    it('should not return disabled items from getEnabledItems_()', async () => {
+      impl.hasTemplate_ = true;
       const sourceData = ['apple', 'mango', 'pear'];
-      const renderedChildren = sourceData.map(item => {
-        const renderedChild = doc.createElement('div');
-        renderedChild.setAttribute('data-value', item);
-        return renderedChild;
-      });
-      renderedChildren[2].setAttribute('data-disabled', '');
+      const rendered = getRenderedSuggestions();
+      rendered.children[2].removeAttribute('data-value', '');
+      rendered.children[2].setAttribute('data-disabled', '');
       env.sandbox
-        .stub(impl.templates_, 'renderTemplateArray')
-        .returns(Promise.resolve(renderedChildren));
+        .stub(impl.ssrTemplateHelper_, 'applySsrOrCsrTemplate')
+        .returns(Promise.resolve(rendered));
 
-      return impl.renderResults_(sourceData, impl.container_).then(() => {
-        expect(impl.container_.children.length).to.equal(3);
-        expect(impl.getEnabledItems_().length).to.equal(2);
-        expect(impl.container_.children[2].hasAttribute('aria-disabled')).to.be
-          .true;
-      });
+      await impl.renderResults_(sourceData, impl.container_);
+      expect(impl.container_.children.length).to.equal(3);
+      expect(impl.getEnabledItems_().length).to.equal(2);
```

Comment:
```suggestion
      expect(impl.getEnabledItems_()).to.have.length(2);
```

There are also a few other places this can be changed.